x-pack/plugins/upgrade_assistant/public/application/components/overview/overview.tsx

```diff
@@ -25,8 +25,9 @@ import { getBackupStep } from './backup_step';
 import { getFixIssuesStep } from './fix_issues_step';
 import { getFixLogsStep } from './fix_logs_step';
 import { getUpgradeStep } from './upgrade_step';
+import { getUpgradeSystemIndicesStep } from './upgrade_system_indices';
 
-type OverviewStep = 'backup' | 'fix_issues' | 'fix_logs';
+type OverviewStep = 'backup' | 'update_system_indices' | 'fix_issues' | 'fix_logs';
```

Comment:
Can we use the term "migrate" instead of "upgrade" or "update" throughout this change? I feel like an upgrade implies a level up, and an update is a minor change. But in this case we're fundamentally changing the nature of these indices which sounds too radical to be characterized as an update, and without a direct user benefit I don't think it's fair to call it an upgrade. @debadair WDYT?

Reply:
Yeah I think you make a very good point, @debadair also seems to be in sync with your thoughts from the copy review she made. I changed the wording of `update/upgrade` to `migrate` in variables, copy and files/folders with https://github.com/elastic/kibana/pull/110593/commits/bb07f52cb92a8f6a2bf10bc83b63e832d194077e